### ISSUE TITLE
Accessibility Violations: The ARIA attribute 'aria-label' is not valid for the element <header> with ARIA role 'none'

### ISSUE THREAD

**Issue by PallaviPattikuntapalli** [IBM/carbon-components-angular#2183]
> Is this issue related to a specific component?

   <Header>
> What did you expect to happen? What happened instead? What would you like to see changed?

    The ARIA attribute 'aria-label' is  valid for the element <header> with ARIA role 'none'
> What browser are you working in?

   Mozilla
> What version of the Carbon Design System are you using?

"carbon-components": "^10.56.0",
"carbon-components-angular": "^4.59.8",

* Screenshots or code
![image](https://user-images.githubusercontent.com/108853103/177743060-a22ac94a-4219-41ce-a93c-d53793464ea2.png)

**Comment by maicongodinho** [comment]
Can you please share your implementation of ibm-header?

**Comment by maicongodinho** [comment]
I was not able to reproduce the problem.

<img width="1378" alt="image" src="https://user-images.githubusercontent.com/94566751/181503271-678a1abf-d74d-4384-a463-7d036b4bfd12.png">

Check this CodeSandbox: https://codesandbox.io/s/carbon-components-angular-2183-wsl66n

The code CodeSandbox package.json:

```
...,
"carbon-components": "10.57.2",
"carbon-components-angular": "4.59.9",
...
```

Can you try to reproduce this in a CodeSandbox app?

**Comment by PallaviPattikuntapalli** [comment]
<ibm-header name="IBM Offering Infomation"></ibm-header>
we have also same code as codesand , once will update to 10.57 and let yo know if we get same issue

**Comment by PallaviPattikuntapalli** [comment]
![image](https://user-images.githubusercontent.com/108853103/181545920-85f5ef96-d324-4a69-b894-7ac27dd41b68.png)

We are also having carbon components latest version , Even though we have added same code  in CodeSandbox, unable to reproduce the violation .

**Comment by PallaviPattikuntapalli** [comment]
Any updates from this issue

**Comment by PallaviPattikuntapalli** [comment]
We are unable to reproduce the issue in CodeSandbox, but still we are getting same issue.

**Comment by sant-santhiya** [comment]
Hi, is there an update on this issue?

**Comment by Akshat55** [comment]
Closing issue since it cannot be reproducible via storybook and codesandbox.
